node/node.go

```diff
@@ -350,11 +350,24 @@ func makeNode(
 		cfg.StateSync.TempDir,
 		nodeMetrics.statesync,
 		eventBus,
-	)
+		// the post-sync operation
+		func(ctx context.Context, state sm.State) error {
+			csReactor.SetStateSyncingMetrics(0)
+
+			// TODO: Some form of orchestrator is needed here between the state
+			// advancing reactors to be able to control which one of the three
+			// is running
+			// FIXME Very ugly to have these metrics bleed through here.
+			csReactor.SetBlockSyncingMetrics(1)
```

Comment:
This suggests to me that these metrics should probably just be in the statesync and blocksync reactors respectively so we don't have to awkwardly set a field on a somewhat unrelated reactor.

Reply:
Yeah, this is reasonable assesment, but this is preexisting so I'm disinclined to detangle it at the moment, but not totally opposed. 

At the same time, I think it's not unreasonable to put the "what kind of syncing mode are we doing" maybe wants to be put in one place, and the consensus reactor might be a reasonable place to centralize this.

Reply:
Also agree but it's fine to address that in a later PR